### ISSUE TITLE
Implement the account UI

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#48]
### Who

- [x] Anonymous visitor
- [x] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

* Register accounts in the Xperience database
* Confirm a pending registration (double opt-in)
* Sign in
* Sign out
* Reset a forgotten password

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #45 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
